GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1193,13 +1186,7 @@ private void ShowChanges(FileStatusItem? item, bool staged)
                 return;
             }
 
-            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
-            {
-                await SelectedDiff.ViewChangesAsync(item, openWithDiffTool: () => OpenWithDiffTool(),
-                    cancellationToken: _viewChangesSequence.Next());
-            }).FileAndForget();
-
-            return;
+            SelectedDiff.InvokeAndForget(() => SelectedDiff.ViewChangesAsync(item, openWithDiffTool: OpenWithDiffTool, cancellationToken: _viewChangesSequence.Next()));
```

Comment:
The injection of a JTF instance should be no big deal: The class `TaskManager` provides / encapsulates the JTF instance.
With the initial implementation, it would look like:
```
_taskManager.InvokeAndForget(SelectedDiff, () => SelectedDiff.ViewChangesAsync(...));
```
With another argument added to the extension method, it can be written as before (preferred):
```
SelectedDiff.InvokeAndForget(() => SelectedDiff.ViewChangesAsync(...), _taskManager);
```

---

Furthermore, I should feed every already available `CancellationToken` to `InvokeAndForget()`, too - but in another follow-up PR:

```
CancellationToken cancellationToken = _viewChangesSequence.Next();
SelectedDiff.InvokeAndForget(() => SelectedDiff.ViewChangesAsync(..., cancellationToken), _taskManager, cancellationToken);
```